ha/resource/sspl

```diff
@@ -111,41 +115,48 @@ stateful_start() {
     stateful_check_state "master"
     if [ $? -eq 0 ]; then
         # CRM Error - Should never happen
+        log "START (master)..."
         systemctl start sspl-ll
+        log "START (master): $?"
         return $OCF_RUNNING_MASTER
     fi
     stateful_update "slave"
     "${HA_SBIN_DIR}/crm_master" -l reboot -v 1
+    log "START... (crm_reboot: $?)"
     systemctl start sspl-ll
+    log "START: $?"
+
     return $OCF_SUCCESS
 }
 
 stateful_demote() {
     stateful_monitor
     sspl_state=`systemctl is-active sspl-ll`
-    if [ $sspl_state != 'active' ]; then
-       systemctl start sspl-ll
-    fi
+    log "DEMOTE: curr_state=$sspl_state"
+    [ $sspl_state == 'active' ] || return $OCF_NOT_RUNNING
+
     stateful_update "slave"
 
     if [ -z $OCF_RESKEY_sspl_mode ]; then
         OCF_RESKEY_sspl_mode="/var/cortx/sspl/data/state.txt"
     fi
 
     echo "state=degrade" > /var/cortx/sspl/data/state.txt
+    /usr/bin/kill -s SIGHUP $sspl_pid
```

Comment:
@cdeshmukh @saurabhkhanvilkar @andriytk 
I have seen at-least on QA setup where both SSPL instances were monitoring the storage enclosure. I think it could happen due to this missing fix in the current SSPL resource agent script. During tests I believe SSPL instances go through Active and Degraded transitions from the pacemaker perspective but I think the Active instance still remain active since this signal sending code is missing. I believe we need to merge this ASAP.